client/index.html

```diff
@@ -787,6 +787,19 @@ <h1 class="title">Help</h1>
 
 						<div class="help-item">
 							<div class="subject">
+								<code>/rejoin</code>
+							</div>
+							<div class="description">
+								<p>
+									Equivalent to doing <code>/part</code> followed immediately
+									by <code>/join #channel</code>.
```

Comment:
I'd prefer something like "Reload the current channel by leaving it then immediately re-joining it."

Reply:
I've been toying with wordings. Do any of these strike your fancy?

````
Quickly part and rejoin the current channel. Effectively
reloads the user list and syncs modes, at the expense of
losing channel scrollback.
----
Shortcut to leave and then immediately rejoin the current
channel, reloading users/modes/etc.
----
Leaves and immediately rejoins the current channel. Useful
to quickly get op from ChanServ in an empty channel, for example.
````

I'm not convinced that the verbiage about losing history is needed, but I think it's helpful unless a way to keep history can be implemented. :\

Reply:
Good stuff @dgw (sorry I missed that notification)! 👏

I like a mix of first and last:

```
Leave and immediately rejoin the current channel. Useful
to quickly get op from ChanServ in an empty channel, for example.
```

> I'm not convinced that the verbiage about losing history is needed

I agree with that part, let's not.